src/library/Uno.Material/Styles/Application/Colors.xaml

```diff
@@ -28,6 +28,10 @@
 			<Color x:Key="OnErrorColor">#000000</Color>
 			<Color x:Key="OverlayColor">#51000000</Color>
 
+			<!-- Needed for android thumbtints -->
+			<Color x:Key="PrimaryDisabledVariantLightColor">#E9E5FA</Color>
```

Comment:
How will the dev override those? Can they be based off exising colors from the palette?

Reply:
These colors can't have an opacity value, therefore the can't be derived from the palette. Maybe design can add them as special colors

Reply:
These colors can't have an opacity value, therefore the can't be derived from the palette. Maybe design can add them as special colors